server/src/routes/callAutomationEvent.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as express from 'express';
+import { CALLCONNECTION_ID_TO_CORRELATION_ID } from '../utils/callAutomationUtils';
+
+const router = express.Router();
+
+router.post('/', async function (req, res) {
+  try {
+    if (req.body[0].type === 'Microsoft.Communication.CallConnected') {
```

Comment:
Avoid undefined reference exception
```suggestion
    if (req.body[0]?.type === 'Microsoft.Communication.CallConnected') {
```